testing/e2e/workload_test.go

```diff
@@ -181,6 +182,10 @@ func TestCreateFromGitWithAnnotations(t *testing.T) {
 				dir, _ := ioutil.TempDir("", "")
 				defer os.RemoveAll(dir)
 
+				if regexProgress.FindString(output) == "" {
+					t.Errorf("unexpected error: expected progressbar in output %v", output)
```

Comment:
repetition of words:
how about `t.Errorf("error: expected progressbar in output %v", output)`
`t.Errorf("expected progressbar in output %v", output)`

Reply:
same for rest of the error msgs in this file

Reply:
Updated